core/iwasm/common/gc/gc_common.c

```diff
@@ -892,6 +892,15 @@ wasm_runtime_pop_local_object_refs(WASMExecEnv *exec_env, uint32 n)
     } while (--n > 0);
 }
 
+WASMLocalObjectRef *
+wasm_runtime_get_current_local_object_ref(WASMExecEnv *exec_env)
```

Comment:
The `current` in function name seems unnecessary. How about rename it as `wasm_runtime_get_local_object_ref`?

Reply:
And do you want to get non-first ref?
e.g.

``` C
wasm_runtime_get_local_object_ref(exec_env, 0);
wasm_runtime_get_local_object_ref(exec_env, 1);
wasm_runtime_get_local_object_ref(exec_env, 2);
...
```

Reply:
`wasm_runtime_get_local_object_ref`  the name is ok. I just need to get the current ref, as I continue to push into ref, when I need to pop, I can continue to move the pointer until it is consistent with the current pointer I obtained before, which means that I have popped all the obj that I pushed before.

Reply:
@xujuntwt95329

Reply:
How about `wasm_runtime_cur_local_object_ref`?

Reply:
`wasm_runtime_get_cur_local_object_ref ` is better, what do you think?

Reply:
It is OK to me. BTW, should we shorten the function names, like:
```C
wasm_runtime_push_local_obj_ref
wasm_runtime_pop_local_obj_ref
wasm_runtime_pop_local_obj_refs
wasm_runtime_get_cur_local_obj_ref
```
@xujuntwt95329

Reply:
> It is OK to me. BTW, should we shorten the function names, like:
> 
> ```c
> wasm_runtime_push_local_obj_ref
> wasm_runtime_pop_local_obj_ref
> wasm_runtime_pop_local_obj_refs
> wasm_runtime_get_cur_local_obj_ref
> ```
> 
> @xujuntwt95329

These names seems better

Reply:
OK, thanks, done

Reply:
@LevelCA should also change wasm_runtime_push_local_object_ref/wasm_runtime_pop_local_object_ref/wasm_runtime_pop_local_object_refs to wasm_runtime_push_local_obj_ref/wasm_runtime_pop_local_obj_ref/wasm_runtime_pop_local_obj_refs, and the related codes in core/iwasm?

Reply:
@wenyongh e. it's my fault.  All have been modified. thks